File: subprojects/example/src/java/com/demo/app/restapi/impl/Status.java
package com.demo.app.restapi.impl;


import java.time.LocalDateTime;

import javax.ws.rs.GET;
import javax.ws.rs.Path;
import javax.ws.rs.Produces;

@Path("status")
public class Status {

    @GET
    @Produces("text/plain")
    public String getStatus() {
    	LocalDateTime now = LocalDateTime.now(); 
        return "This is active Java code with server time " + now + " source from 10:11";
    }
}

File: subprojects/test/src/kotlin/software/hsharp/demo/impl/Status.kt
package software.hsharp.demo.impl

import javax.ws.rs.GET
import javax.ws.rs.Path
import javax.ws.rs.Produces
import java.time.LocalDateTime

@Path("status")
class Status {

    val status: String
        @GET
        @Produces("text/plain")
        get() = "This is active Kotlin code with server time " + LocalDateTime.now() + " source from 10:03";
}
